src/smallUsefulFunctions.cpp

```diff
@@ -236,11 +236,51 @@ QString textToLatex(const QString &text)
 	replaceList.append(QPair<QString, QString> ("~", "\\~{}"));
 	replaceList.append(QPair<QString, QString> ("_", "\\_"));
 	replaceList.append(QPair<QString, QString> ("^", "\\^{}"));
+	replaceList.append(QPair<QString, QString> ("\u0393", "$\\Gamma$"));

```

Comment:
There are people in the world who use greek letters. This replacements does not make sense for someone who wants to put in a greek filename for example.
Also starting a math env for every letter is not useful.
So, the whole greek letter part is not useful.

Reply:
I'll remove it